lib/process-services-cloud/src/lib/form/services/form-definition-selector-cloud.service.spec.ts

```diff
@@ -0,0 +1,101 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { TestBed } from '@angular/core/testing';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { AlfrescoApiService, CoreModule, setupTestBed, AppConfigService } from '@alfresco/adf-core';
+import { FormDefinitionSelectorCloudService } from './form-definition-selector-cloud.service';
+
+declare let jasmine: any;
+
+const responseBody = [
+    {
+        formRepresentation: {
+            id: 'form-de8895be-d0d7-4434-beef-559b15305d72',
+            name: 'Form 1',
+            description: '',
+            version: 0,
+            standAlone: true
+        }
+    },
+    {
+        formRepresentation: {
+            id: 'form-de8895be-d0d7-4434-beef-fgr34ttgrtgd',
+            name: 'Form 2',
+            description: '',
+            version: 0,
+            standAlone: false
+        }
+    },
+    {
+        formRepresentation: {
+            id: 'form-de8895be-d0d7-4434-beef-53453453452',
+            name: 'Form 3',
+            description: '',
+            version: 0
```

Comment:
is there a scenario possible in reality that there is a response json coming back with standAlone variable but no value assigned to it? or does it always have a default value? 
if so then what is the default flag..

Reply:
the check in the filter on undefined added by David should save this scenario